src/air/tags/constants.py

```diff
@@ -115,3 +115,21 @@ class TagKeys:
     """,
     re.IGNORECASE | re.DOTALL | re.VERBOSE,
 )
+_LOOKS_LIKE_HTML_UNICODE_RE: Final = re.compile(
```

Comment:
Any chance this could be targeted with an XML bomb attack?

I know this is for converting HTML to Tags, but still I wonder about the safety of this.